src/components/text/Link.jsx

```diff
@@ -62,7 +62,7 @@ const Link = (props: LinkPropsType) => {
   if (!href) {
     return (
       <Text
-        type={LINK_TYPE.SPAN}
+        type="span"
```

Comment:
What do we gain by this change?

Reply:
in the newest Flow and VS Code new version can be auto-suggested, while the first one needs to be remembered :)